plugins/modules/proxmox_storage_list.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright Julian Vanden Broeck (@l00ptr) <julian.vandenbroeck at dalibo.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = """
+---
+module: proxmox_storage_list
+short_description: List content from a Proxmox VE storage
+version_added: 8.2.0
+description:
+  - Retrieves information about stored objects on a specific storage attached to a node.
+attributes:
+  check_mode:
+    support: none
+  diff_mode:
+    support: none
+options:
+  storage:
+    description:
+      - Only return content stored on that specific storage.
+    aliases: ['name']
+    type: str
+    required: true
+  node:
+    description:
+      - Proxmox node to which the storage is attached.
+    type: str
+    required: true
+  content:
+    description:
+      - Filter on a specific content type. Should be one of those value backup, rootdir, images, iso.
```

Comment:
The values are listed directly next to this description, so there's no need to list them again :)
```suggestion
      - Filter on a specific content type.
```